recipe/bld.bat

```diff
@@ -32,3 +32,5 @@ if errorlevel 1 exit 1
 :: Install.
 nmake install
 if errorlevel 1 exit 1
+
+del %LIBRARY_PREFIX%\*.txt %LIBRARY_PREFIX%\COPYING
```

Comment:
I would leave the `COPYING` file. Even though conda has that in the package metadata some users do look for it in the "software designated folder."